cli/options.go

```diff
@@ -334,6 +335,8 @@ func ProjectFromOptions(options *ProjectOptions) (*types.Project, error) {
 
 	options.loadOptions = append(options.loadOptions, withNamePrecedenceLoad(absWorkingDir, options))
 
+	options.loadOptions = append(options.loadOptions, withConvertWindowsPaths(options))
```

Comment:
This can be added in the line above to use just one append in the section

Reply:
yeah! how does it look like now?